### PR TITLE
[DRAFT] Backporting yarn 1.16.2 to 1.16.1

### PR DESCRIPTION
**Pull request by LambdAurora**
The backporting PRs are back.

All the related files to backporting are here: https://gist.github.com/LambdAurora/04608dac389e91cb5c41553641f6b8d2

This is highly WIP for now.

### CODE REVIEW

File: mappings/net/minecraft/client/realms/gui/screen/RealmsResetWorldScreen.mapping [hunk 1]
```diff
@@ -22,7 +22,6 @@ CLASS net/minecraft/class_4410 com/mojang/realmsclient/gui/screens/RealmsResetWo
 	METHOD <init> (Lnet/minecraft/class_437;Lnet/minecraft/class_4877;Lnet/minecraft/class_2561;Lnet/minecraft/class_2561;ILnet/minecraft/class_2561;Ljava/lang/Runnable;Ljava/lang/Runnable;)V
 		ARG 1 parent
 	METHOD method_21370 drawFrame (Lnet/minecraft/class_4587;IILnet/minecraft/class_2561;Lnet/minecraft/class_2960;ZZ)V
-		ARG 1 matrices
```
Comment (ghost) [inline mappings/net/minecraft/client/realms/gui/screen/RealmsResetWorldScreen.mapping]: This should probably be re-added as this is a MatrixStack.

File: mappings/net/minecraft/client/render/SkyProperties.mapping [hunk 2]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_5294 net/minecraft/client/render/SkyProperties
-	FIELD field_24609 BY_DIMENSION_TYPE Lit/unimi/dsi/fastutil/objects/Object2ObjectMap;
```
Comment (ghost) [inline mappings/net/minecraft/client/render/SkyProperties.mapping]: This old name was correct at the time.

Comment (LambdAurora) [reply mappings/net/minecraft/client/render/SkyProperties.mapping]: Fixed

**Comment by ghost** [comment]
Looks good other than the few comments.

**Review by YanisBft — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by liach** [comment]
Should be good. I just recall vanilla has mechanism change for tags or maybe some dimension/biome loading stuff; maybe some names there for 1.16.2 doesn't fit in 1.16.1.

**Comment by LambdAurora** [comment]
Yes, that's why it's a draft for now, I still have to figure out what should not be backported.

File: mappings/net/minecraft/block/ChainBlock.mapping [hunk 3]
```diff
@@ -1,3 +1,3 @@
 CLASS net/minecraft/class_5172 net/minecraft/block/ChainBlock
-	FIELD field_23986 SHAPE Lnet/minecraft/class_265;
+	FIELD field_23986 Y_SHAPE Lnet/minecraft/class_265;
```
Comment (haykam821) [inline mappings/net/minecraft/block/ChainBlock.mapping:2]: This is not applicable as chains only have one direction (on the Y axis) in 1.16.1.

**Review by haykam821 — CHANGES_REQUESTED** [review]

**Comment by modmuss50** [comment]
Is this still WIP/Draft?

**Comment by haykam821** [comment]
Definitely do not merge this yet since the `field_23986` issue is unresolved.

**Comment by YanisBft** [comment]
@LambdAurora do you still want to maintain this?

**Pull request closed without merging**